journal-of-sport-and-health-science.csl

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
```

Comment:
> I'd typically just fix these myself, but since you're intending to do more -- thanks again! -- it makes sense to send it back to you for future reference

Sure, I prefer it this way so that I can learn de subtleties as I come across different styles. `default-locale` added.